instrumentation-core/spring/spring-boot-autoconfigure/spring-boot-autoconfigure.gradle

```diff
@@ -31,8 +32,15 @@ dependencies {
   implementation 'org.springframework.boot:spring-boot-starter-webflux'
   implementation project(':instrumentation-core:spring:spring-webflux-5.0')
   
-  api deps.opentelemetryApi 
-  api "io.opentelemetry:opentelemetry-exporters-logging:0.5.0"
```

Comment:
Moving these apis to an opentelemetry-spring-starter. A draft of this can be see here: https://github.com/mabdinur/opentelemetry-java-instrumentation/pull/6